source/ide/config/cfg_methods.bas

```diff
@@ -320,15 +320,22 @@ SUB ReadInitialConfig
         WriteConfigSetting displaySettingsSection$, "IDE_SortSUBs", "False"
     END IF
 
+    tmpKwCap = _FALSE: tmpKwLow = _FALSE
     IF ReadConfigSetting(displaySettingsSection$, "IDE_KeywordCapital", value$) THEN
```

Comment:
Instead of adding IDE_KeywordLowercase, can't we have
IDE_KeywordCapital = -1 ' upper case
IDE_KeywordCapital = 0 ' camel case
IDE_KeywordCapital = 1 ' lower case